community/modules/scripts/spack-setup/README.md

```diff
@@ -119,6 +119,46 @@ sudo -i spack python -m pip install package-name
 [SPACK_PYTHON]: https://spack.readthedocs.io/en/latest/getting_started.html#shell-support
 [builds]: https://spack.readthedocs.io/en/latest/binary_caches.html
 
+## Spack Permissions
+
+By default, only root will be have permission to install spack packages. To
+manually call Spack commands as root use the `sudo -i` command, for example:
+`sudo -i spack install tar`. The `-i` runs the command as a login shell and is
+required to make sure `spack` will be the `PATH`.
+
+Installed packages should be readable/executable by anyone on the VM.
+
+### Non-root package installation
+
+If there is a need to have a non-root user to install spack packages it is
+recommended to create a separate installation for that user and chain Spack installations
+([Spack docs](https://spack.readthedocs.io/en/latest/chain.html#chaining-spack-installations)).
+
+Steps to chaining Spack installations:
+
+1. Clone a new spack installation:
+
+    ```sh
+    git clone https://github.com/spack/spack.git <new_location>/spack
+    ```
+
+2. Point the new Spack installation to the system Spack installation. Create a
+   file at `<new_location>/spack/etc/spack/upstreams.yaml` with the following
+   contents:
+
+    ```yaml
+    upstreams:
+      spack-instance-1:
+        install_tree: /sw/spack/opt/spack/
+    ```
+
+3. Add the following line to your `.bashrc` to make sure the new `spack` is in
+   your `PATH`.
+
+   ```sh
+   . <new_location>/spack/share/spack/setup-env.sh
```

Comment:
Has this been manually verified to work after the `/etc/profile.d` script activates the "upstream/system" Spack installation?

Reply:
yes. I tested on spack installation using our spack-setup modules.

Reply:
I see that in description now (not in original notification)